src/components/ActionMenu/components/Actions/Actions.tsx

```diff
@@ -72,12 +153,26 @@ export function Actions({actions = [], groups = []}: Props) {
   });
 
   return (
-    <div className={styles.ActionsLayout}>
+    <div className={className} ref={actionsLayoutRef}>
       {newDesignLanguage ? (
-        <ButtonGroup>{actionMarkup}</ButtonGroup>
+        <ButtonGroup segmented>
+          {showableActions && showableActions.length > 0
+            ? showableActions.map((action) => (
```

Comment:
I meant to extract this out to a constant but I will do that in a follow up commit